docs/introduction_tutorial/intro_tutorial_part2.ipynb

```diff
@@ -0,0 +1,1228 @@
+{
```

Comment:
<p>Perhaps add a note like this</p><p> </p><blockquote>If we wanted to run the recipe on the full dataset (as opposed to the much smaller pruned version), we would just repeat the above steps on <code>recipe</code> rather than <code>recipe_pruned</code>. This would take a long time, but it would work.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pangeo-forge/pangeo-forge-recipes/pull/293/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>